### PR TITLE
Dashboard -mycandidates

### PR DESCRIPTION
**Pull request by bulganerdenebaatar**
Added mycandidates table but not yet responsive. Candidate view component created. 
Screen Shot 2021-07-06 at 8 40 51 PM

https://user-images.githubusercontent.com/1626382/127606178-cc84090a-82ed-4748-8974-7a0ce3015cbf.mp4

### CODE REVIEW

File: web/src/components/Admin/Admin.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* .fixedHeight {
```
Comment (ulziibay) [inline web/src/components/Admin/Admin.css]: can we just remove this file?

File: web/src/components/Admin/Admin.js [hunk 2]
```diff
@@ -1,16 +1,25 @@
-import React from 'react';
-import classes from '../ReferrerForm/Referrer.css';
-import Admin from './AdminForm/Admin';
-import {Route} from 'react-router-dom';
+import React from "react";
+// import { Route } from "react-router-dom";
```
Comment (ulziibay) [inline web/src/components/Admin/Admin.js]: Remove this if you do not need it

File: web/src/components/Admin/AdminComponents/Sidebar/Sidebar.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    min-height: 150vh; 
+    box-shadow: 2px 2px 3px 2px #dddbdb;
+}
+
+
+/* .button {
+    padding: 10px;
+    background-color: white;
+    border-radius: 25%;
+} */
```
Comment (ulziibay) [inline web/src/components/Admin/AdminComponents/Sidebar/Sidebar.css:21]: Remove this if you do not need it

File: web/src/components/Admin/AdminComponents/Sidebar/Sidebar.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+//       >
+//         <button onClick={() => this.onSetSidebarOpen(true)} className={classes.button}>
+//           <ArrowForwardIosIcon />
+//         </button>
+//       </Sidebar>
+//     );
+//   }
+// }
+
+// export default SidebarComponent;
```
Comment (ulziibay) [inline web/src/components/Admin/AdminComponents/Sidebar/Sidebar.js]: Remove this if you do not need it

File: web/src/components/Admin/AdminComponents/Topbar/SearchBar.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+        cancelOnEscape={true}
+        style={{ width: "150px" }} 
+    />
+    // <div className={classes.search}>
+    //    <div className={classes.searchInputs}>
+    //        <input className={classes.input} type="text" placeholder="Search" />
+    //        <div className={classes.searchIcon}> <SearchIcon /> </div>
+    //    </div>
+    //    {/* <div className="dataResult"></div> */}
+    // </div>
```
Comment (ulziibay) [inline web/src/components/Admin/AdminComponents/Topbar/SearchBar.js]: Remove this if you do not need it

File: web/src/components/Admin/AdminComponents/Topbar/Topbar.js [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        // <React.Fragment>
+        //     <div className={classes.container}>
+        //       <div className={classes.title}>
+        //       <p>MITPU Applicant Tracking System</p>
+        //       </div>
+        //       <div className={classes.search}>
+        //         <SearchBar />
+        //       </div>
+        //     </div>
+        // </React.Fragment>
```
Comment (ulziibay) [inline web/src/components/Admin/AdminComponents/Topbar/Topbar.js]: Remove this if you do not need it

File: web/src/hoc/LayoutTwo/Admin.js [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+//             <Grid item xs={12}>
+//               <Paper className={classes.fixedHeight}>
+//                EMAIL
+//               </Paper>
+//             </Grid>
+//           </Grid>
+//         </Container>
+//       </main>
+//     )
+// }
```
Comment (ulziibay) [inline web/src/hoc/LayoutTwo/Admin.js]: Remove this if you do not need it

**Review by ulziibay — CHANGES_REQUESTED** [review]
Overall looks great. Two asks:

1) Can you remove the commented out code? It makes it easier to review and maintain the code
2) Can you add a screenshot or video of the interaction between the main dashboard and the candidate view? In the PR body, I only see the dashboard, not the candidate view.

**Comment by bulganerdenebaatar** [comment]
Hi! I removed all of the uncommented parts in the files. Also, I wasn’t sure what you meant by your ask #2…



Thanks!

**Comment by ulziibay** [comment]
> Hi! I removed all of the uncommented parts in the files

Please update this branch with your latest commits so that those changes are reflected.  As is, it is not shown in this PR.

> Also, I wasn’t sure what you meant by your ask #2

Can you add a screenshot or video of how the candidate view section looks/works at the moment?

**Comment by bulganerdenebaatar** [comment]
I just committed the changes and pushed it. I also sent you the video of the candidate view on Slack. Wasn’t sure where you wanted me to send you so I will send the video on here, too.


Thanks!

**Review by ulziibay — APPROVED** [review]
Just tried it locally, looks great. Thanks for pushing both features in!

File: core/src/main/resources/application-dev.properties [hunk 8]
```diff
@@ -1,8 +1,8 @@
 ################################################################################
 # Configuration for development environment
 # Please set below properties!
-spring.datasource.url=jdbc:mysql://${host}/${schema_name}
+spring.datasource.url=jdbc:mysql://referral-1.cdaj903rkno2.us-west-1.rds.amazonaws.com/referral
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.datasource.username=${user_name}
-spring.datasource.password=${password}
+spring.datasource.username=referral
+spring.datasource.password=Referral&2020
```
Comment (ulziibay) [inline core/src/main/resources/application-dev.properties]: Actually can you undo this and add this file to the .gitignore file? This credential is not meant to be pushed to github!

**Review by ulziibay — CHANGES_REQUESTED** [review]
can you remove the DB credentails?

**Comment by bulganerdenebaatar** [comment]
Yes, I will work on it right now. Have to google how to undo a git commit or git push.

**Comment by ulziibay** [comment]
> Yes, I will work on it right now. Have to google how to undo a git commit or git push.

You can just create another commit that removes the changes to this file for now. Then, once "approved", you can select the "squash and merge" option.

**Review by ulziibay — APPROVED** [review]
Looks good!

**Pull request merged**